frontend/src/components/product/Product.js

```diff
@@ -1,10 +1,17 @@
-import React, { useState } from 'react'
+import React, { useState, useContext } from 'react'
 import "./Product.css"
-import { AiFillHeart, AiOutlineHeart,AiFillStar } from "react-icons/ai"
-function Product({ productname, productimage, productprice, originalprice, producttag, productrating }) {
+import { AiFillHeart, AiOutlineHeart, AiFillStar } from "react-icons/ai"
+import { CartContext } from "../../context/CartContext"
+import { ToastContainer } from "react-toast"
+function Product({ productid,productname, productimage, productprice, originalprice, producttag, productrating }) {
+
   const [addwishlist, setAddwishlist] = useState(false)
+  const { addTocart } = useContext(CartContext)
+  const product = {productid,productname, productimage, productprice, originalprice, producttag, productrating}
+
```

Comment:
Please use camel case properly to name variables you can use productId, productName, etc.

Reply:
Thanks fro your suggestion @bhushannaikaroskar , I will change the names into camel case.